src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    result[robot[key]] = key;
```

Comment:
it's better to check here if `result[robot[key]]` already exists and return `null`